### DISCUSSION TITLE
GitHub Actions has encountered an internal error when running your job.

### DISCUSSION THREAD

**Discussion by corneliusellen** [community/community#36150] [Actions]
Getting this error with no further explanation. I just started using ArtiomTr/jest-coverage-report-action@v2.1.1 and marocchino/sticky-pull-request-comment@v2 in my `on-pull-request.yaml`, but it's unclear if this is what is causing the job to be immediately cancelled.

The run is here: https://github.com/enzyme-health/wheel-api/actions/runs/3250906684

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @corneliusellen! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
